x-pack/plugins/observability_solution/observability_onboarding/e2e/playwright/playwright.config.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import dotenv from 'dotenv';
+import { defineConfig, devices } from '@playwright/test';
+import path from 'path';
+import { log } from './lib/logger';
+import { assertEnv } from './lib/assert_env';
+
+const dotEnvPath = process.env.DOTENV_PATH ?? path.join(__dirname, '.env');
+
+dotenv.config({ path: dotEnvPath });
+
+assertEnv(process.env.ARTIFACTS_FOLDER, 'ARTIFACTS_FOLDER is not defined.');
+
+export const STORAGE_STATE = path.join(__dirname, process.env.ARTIFACTS_FOLDER, '.auth/user.json');
+
+// eslint-disable-next-line import/no-default-export
+export default defineConfig({
+  testDir: './',
+  outputDir: './.playwright',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
```

Comment:
I see you'd like it to be fully parallel, which I believe is at the worker scope. 
Is it ok that below workers is set to `1`?

I guess I'm thinking how can it be fully parallel with one worker.

Reply:
Good point! Tbh, I did not write this config in the first place (migrated from the `oblt-playwright` repo), but I see this is the recommended basic config from the Playwright docs. I think it makes sense though in our use case, because tests are run both on the CI and locally, so there will be a single worker for the CI (AFAIK the instance running it does not have a lot of resources) and locally it will be the default value, which probably calculated depending on CPU cores count.